docs/xroad_security_server_toolkit_user_guide.md

```diff
@@ -88,6 +89,7 @@ Doc. ID: XRDSST-CONF
             * [4.2.5.4 Certificate activation](#4254-certificate-activation)
             * [4.2.5.5 Download internal TSL certificates](#4255-download-internal-tsl-certificates)
             * [4.2.5.6 List certificates](#4256-list-certificates)
+            * [4.2.5.6 Disable certificates](#4256-disable-certificates)
```

Comment:
Please update the chapter number as well as it is currently exactly a copy-paste of "list certificates"